unable to link nodecloud-gcp-plugin and nodecloud

## Expected Behavior
Getting an error saying package nodecloud-gcp-plugin not found while linking nodecloud and nodecloud-gcp-plugin

## Actual Behavior
```
$ npm link nodecloud-gcp-plugin@1.0.0
npm ERR! code E404
npm ERR! 404 Not Found: nodecloud-gcp-plugin@1.0.0

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<user>\AppData\Roaming\npm-cache\_logs\2019-02-16T05_41_23_158Z-debug.log
```

## Steps to Reproduce the Problem

  1. git clone https://github.com/cloudlibz/nodecloud-gcp-plugin
  2. cd nodecloud-gcp-plugin
  3. npm link
  4. git clone https://github.com/cloudlibz/nodecloud
  5. cd nodecloud
  6. npm link nodecloud-gcp-plugin

## Specifications

  - Version: node v8.13.0
  - Platform: Windows 10
  - Subsystem:

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "nodecloud-gcp",
+  "name": "nodecloud-gcp-plugin",
   "version": "1.0.0",
   "description": "GCP Wrapper for NodeCloud Core",
   "main": "index.js",

